src/utils/components/AddBootableVolumeModal/components/SourceTypeSelection/SourceTypeSelection.tsx

```diff
@@ -31,6 +33,10 @@ const SourceTypeSelection: FC<SourceTypeSelectionProps> = ({
       event.preventDefault();
       setFormSelection(value);
       setIsOpen(false);
+
+      if (value !== DROPDOWN_FORM_SELECTION.USE_SNAPSHOT) {
+        resetDiskSize();
```

Comment:
why we need this exception? it leads to confusing behaviour i.e.
1. choose "volume" source and modify the disk size to X
2. switch to "snapshot" source - the disk size is X but cannot be changed